BookingsApi/BookingsApi.IntegrationTests/Database/Queries/GetJusticeUsersListQueryHandlerDatabaseTests.cs

```diff
@@ -23,7 +23,7 @@ public void Setup()
         public async Task Should_return_null_when_no_users_are_found()
         {
             await Hooks.ClearAllJusticeUsersAsync();
```

Comment:
the test name and assertion doe not match. which one is correct?

Reply:
changed description to have an empty list